src/lib/components/Pagination.svelte

```diff
@@ -0,0 +1,140 @@
+<script context="module" lang="ts">
+	const PAGE_SIZE = 50
```

Comment:
50 feels like a lot.. I tried in the [preview page](https://deploy-preview-12--duders-zone.netlify.app/shows/quick-looks/) and it results in a lot of scrolling. Perhaps we should start with 25 instead?

Alternatively, do we maybe want to have the page size be a parameter that gets passed into the component? Then we would have to pass it into the `paginate` function as well, which seems less than ideal 😕

Reply:
I've changed it to 25.

I don't think passing the page size is any less ideal than passing the page number to both the function and the component.

When you say "be a parameter" do you mean have a dropdown on the page for the user to change it, and add it to the URL?

Reply:
I just meant that we can pass it in to the component itself, like this:
```html
<Pagination totalResults={data.videos.length} currentPage={pageNumber} pageSize={25} />
```
But then we would have to pass it into the `paginate` function as well:
```
$: paginatedVideos = paginate(pageNumber, 25, data.videos)
```
It would make the pagination more versatile, but it seems like a hassle, so let's just leave it the way it is for now.

Reply:
Gotcha. It wouldn't be so bad as a const defined in the page, but yeah we can figure it out later.